DayrSuryan/DSEthiop7.xml

```diff
@@ -118,7 +141,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
           ኢየሱስ᎓ ክርስቶስ᎓ ነአ᎓ ነአ፨ ጊዮርጊስ᎓ እንተ᎓ ትሰርር᎓ በፈረስ᎓</incipit>
                      <explicit xml:lang="gez"><hi rend="rubric">ጊዮር᎓
                      </hi>ሰማዕት᎓ ፍጡነ᎓ ረድኤት᎓ ሐውፀነ᎓ በምኅረት᎓ ወተራድአነ᎓ በቅጽበት᎓᎓
-            ተፈጸመ᎓ በዕለተ᎓ ሠሉስ᎓ አዚ᎓ መጽሐፍ᎓ ናይ᎓ በርሄ᎓ እዩ᎓᎓</explicit>
+            </explicit>
+                     <colophon xml:id="coloph1" xml:lang="ti">
+                        ተፈጸመ᎓ በዕለተ᎓ ሠሉስ᎓ አዚ᎓ መጽሐፍ᎓ ናይ᎓ <persName ref="PRS12557Barha">በርሄ</persName>᎓ እዩ᎓᎓
```

Comment:
እዚ?

Reply:
I don't know, we don't have the images...

Reply:
Write እዚ, it is 85% like this. እዚ is meant in any case.